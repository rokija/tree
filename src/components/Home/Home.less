.Home {
   padding: 0 40px;
}
