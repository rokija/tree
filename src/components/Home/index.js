import React from 'react'
import './Home.less'

const Home = () => <h1>Hello</h1>

export default Home
