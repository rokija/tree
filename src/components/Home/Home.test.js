import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from '.'

describe('Home', () => {
  const wrapper = shallow(<Home />)
  it('renders properly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
