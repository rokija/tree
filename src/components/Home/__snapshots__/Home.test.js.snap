// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders properly 1`] = `
<div
  className="Home"
>
  <input
    onChange={[Function]}
  />
  <button
    onClick={[Function]}
  >
    Add
  </button>
</div>
`;
